english/pinch/jacket/eatit/eatit.md

```diff
@@ -1 +1 @@
-You eat a hole threw the Marshmallow wall. Looking through the hole you see a hallway.
\ No newline at end of file
+You eat a hole threw the Marshmallow wall. Looking through the [hole](hole/hole.md) you see a [hallway](hallway/hallway.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.